src/status_im2/contexts/activity_center/notification/reply/view.cljs

```diff
@@ -37,18 +37,31 @@
 
     nil))
 
+(defn swipeable
```

Comment:
FYI: This piece of code is very similar to 2 other namespaces, but I prefer to tackle this duplication in a follow up PR, when other swipeables for *notifications with call to action* will be implemented. I expect to see some differences, so that's why I'm avoiding keeping the code extremely DRY.